src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletControllerTest.java

```diff
@@ -104,5 +129,54 @@ void shouldThrowAnExceptionWhenNameContainsIllegalLetters() throws Exception {
         result.andExpect(status().isBadRequest());
     }
 
+    @Test
+    @DisplayName("Should return empty list of wallets and http ok status")
+    void shouldReturnEmptyList() throws Exception {
+        // when
+        MockHttpServletResponse result = mockMvc.perform(get("/api/wallet")
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().is2xxSuccessful())
+                .andReturn()
+                .getResponse();
+
+        // then
+        assertThat(result.getStatus()).isEqualTo(200);
+        assertThat(result.getContentAsString()).isEqualTo("[]");
```

Comment:
Same here.

Reply:
- [ ]  correct tests to assert http status instead of its numerical value

Reply:
Used `Collections.emptyList().toString()` instead of `[]` String